ProjectTemplates/VisualStudio2015/WindowsUniversal10.VB/WindowsUniversal.VB.vbproj

```diff
@@ -11,7 +11,7 @@
     <DefaultLanguage>en-US</DefaultLanguage>
     <TargetPlatformIdentifier>UAP</TargetPlatformIdentifier>
     <TargetPlatformVersion>10.0.10586.0</TargetPlatformVersion>
```

Comment:
Same as above, either target 16299 or 14393 as in the other project files below.